test/github-issues/134/issue-134.ts

```diff
@@ -0,0 +1,44 @@
+import "reflect-metadata";
+import { createTestingConnections, closeTestingConnections, reloadTestingDatabases } from "../../utils/test-utils";
+import { Connection } from "../../../src/connection/Connection";
+import {Post} from "./entity/Post";
+import { expect } from "chai";
+
+describe("github issues > #134 Error TIME is converted to 'HH-mm' instead of 'HH:mm", () => {
+
+    let connections: Connection[];
+    before(async () => connections = await createTestingConnections({
+        entities: [__dirname + "/entity/*{.js,.ts}"],
+        schemaCreate: true,
+        dropSchemaOnConnection: true,
+    }));
+    beforeEach(() => reloadTestingDatabases(connections));
+    after(() => closeTestingConnections(connections));
+
+
+    it("should successfully persist the post with creationDate in HH:mm and return persisted entity", () => Promise.all(connections.map(async connection => {
+
+        const postRepository = connection.getRepository(Post);
+        const post = new Post();
+        const currentDate = new Date();
+        post.title = "Hello Post #1";
+        post.creationDate = currentDate;
+
+
+        const savedPost = await postRepository.persist(post);
+        const loadedPost = await connection.entityManager
+            .createQueryBuilder(Post, "post")
+            .where("post.id=:id", { id: savedPost.id })
+            .getOne();
+
+
+        expect(post).not.to.be.empty;
+        expect(loadedPost).not.to.be.empty;
+        expect(loadedPost).to.be.eql(savedPost);
```

Comment:
The problem here that `creationDate` must be a string, not a `Date` object. There is a Date support in `time` column type, however it exist there just like an extra feature.

When you load a post, `creationDate` will be returned as a `string`, not `Date` object. Because column type = `time` is converted by orm to string, not a Date. Thats why 

In your assertion you are checking if loadedPost is equal to savedPost, however they aren't equal. Because typeof `savedPost.creationDate` === Date, and in `loadedPost.creationDate` === "string".

You should make a small correction in your test - you should assert only loadedPost.creationDate and check if its equal to `savedPost.creationDate.getDate() + "-" + savedPost.creationDate.getMonth()  + "-" + savedPost.creationDate.getYear();`. Something like that.

Also note that if you want support your `creationDate` Date object, you'll also need it to support a `string` type, because otherwise it wont be type-safe. Something like that:

`creationDate: Date|string`